.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
should we also include the java deps?

Reply:
I did not even realize this was an option 🤯 Let's give it a shot